box-content-sdk/src/main/java/com/box/androidsdk/content/requests/BoxRequestsShare.java

```diff
@@ -406,4 +406,44 @@ protected void onSendCompleted(BoxResponse<BoxCollaboration> response) throws Bo
         }
     }
 
+    /**
+     * Request for updating owner of a collaboration
+     */
+    public static class UpdateOwner extends BoxRequest<BoxVoid, UpdateOwner> {
+
+        private static final long serialVersionUID = 8123965031239671597L;
+
+
+        private String mId;
+
+        /**
+         * Creates a request to update the collaboration with the default parameters
+         *
+         * @param collaborationId id of the collaboration to update
+         * @param requestUrl      URL of the update collaboration endpoint
+         * @param session         the authenticated session that will be used to make the request with
+         */
+        public UpdateOwner(String collaborationId, String requestUrl, BoxSession session) {
```

Comment:
Is there a reason for having this request? Is the UpdateCollaboration request not sufficient?